migrations/20181002234013_create_user_joind_in_username_column.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2018 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+use Cartalyst\Sentinel\Users\EloquentUser;
+use Phinx\Migration\AbstractMigration;
+
+class CreateUserJoindInUsernameColumn extends AbstractMigration
+{
+    /** @var Capsule $capsule */
+    public $capsule;
+
+    public function bootEloquent()
+    {
+        $adapter       = $this->getAdapter()->getAdapter();
+        $options       = $adapter->getOptions();
+        $this->capsule = new Capsule();
+        $this->capsule->addConnection([
+            'driver'   => 'mysql',
+            'database' => $options['name'],
+        ]);
+        $this->capsule->getConnection()->setPdo($adapter->getConnection());
+        $this->capsule->bootEloquent();
+        $this->capsule->setAsGlobal();
+    }
+
+    public function up()
+    {
+        // Create joindin_username
+        $this->table('users')
+            ->addColumn('joindin_username', 'string', ['null' => true])
+            ->update();
+
+        // Go through each record in user, strip out (https://joind.in/user/) and copy to joindin_username
+        $joindin_regex = '/^https:\/\/joind\.in\/user\/(.{1,100})$/';
+
+        $users = EloquentUser::all();
```

Comment:
Personally, I'm not a fan of using an ORM in database migrations, as when the migration is run

* a model or entity may not even exist anymore
* a model or entity might not even work with the schema at the state before the migration is run

Now, since I haven't used Laravel at all, and apparently models have been used in migrations within this project before, so I'm not going to make an issue out of it, but for the record, I would prefer not to use models or entities in a database migration.

* at the time the migration is run,

Reply:
We are already using Eloquent in other migrations, so I do not have a big problem with it.